docs/decisions/0008-error-logging-on-send.rst

```diff
@@ -0,0 +1,26 @@
+8. Error logging on send
+########################
+
+Status
+******
+
+Accepted
+
+Context
+*******
+
+There are numerous strategies for error handling in Kafka, variously optimized for different goals (consistency, reliability, etc.). Some error-handling is built in to the confluent-kafka library, such as automatic retries when producing events. Other errors may occur outside the Kafka code, for example when trying to extract an event key from an event data dictionary.
+
+Decision
+********
+
+When dealing with errors in sending events to the event bus, we will prioritize ensuring that no events are irretrievably dropped. To that end, we will catch exceptions and log the full event data in the error message in a consistent, parseable way.
+
+We will defer any idea of using a retry queue. Also, note that this decision does not cover how we will handle errors on the consumer.
```

Comment:
I think we should make that decision now so we don't have an ambiguous ADR lying around until we figure out consuming. If we decide this is the ADR for both, we should update the title and make it provisional. Otherwise, I think we should leave it as accepted.

Reply:
I'll make it provisional and update the A/C on the consuming issue to make updating the ADR part of it

Reply:
[fixed] though @rgraber should still update the consuming issue